### DISCUSSION TITLE
Home assistant si avvia ma non visualizzo pagina web

### DISCUSSION THREAD

**Discussion by mkel84** [indomus/forum#1269] [HUB: Home Assistant]
Salve,
ho aggiornato all'ultima versione di home assistant ma dal 2 gennaio non riesco più ad accedere al sito web di home assistant, precisamente la sera ha funzionato tutto correttamente mentre la mattina non sono riuscito più ad accedere.
Il servizio di home assistant sembra funzionare correttamente, a parte l'errore sul bluetooth che ho da quando ho aggiornato la versione di python dalla 3.8 alla 3.9.

![image](https://user-images.githubusercontent.com/69473831/148557844-e9ac6b22-1f4f-4da7-9be4-e14be1feb9b3.png)

Secondo voi cosa potrebbe essere? Ho provato anche a lanciare il comando netstat -an | grep 8123 sul raspberry ma non mi da alcun risultato.

**Comment by snakuzzo** [comment]
non devi guardare lì...devi guardare il file di log

**Reply by mkel84** [reply]
Tramite il comando sudo journalctl -f -u home-assistant@homeassistant?

**Reply by snakuzzo** [reply]
no...vai a leggere proprio il file.

Mettiti in tail sul file di log...

`tail -f home-assistant.log`

e riavvia Home Assistant.

Vedi cosa logga

**Reply by mkel84** [reply]
![Cattura](https://user-images.githubusercontent.com/69473831/148571528-d4648e7a-2948-423a-8c7b-cd7af83a60e6.JPG)

**Reply by mkel84** [reply]
ho aggiornato i permessi per le cartelle di home assistant tramite il comando sudo chmod -R 0777 /home/homeassistant e adesso va, grazie mille
